packages/table-schema-provider/src/Features.ts

```diff
@@ -0,0 +1,159 @@
+import { propOr, pathOr } from 'ramda';
+import { TableSchema, tablesListSelectors, tableSelectors, tableFieldSelectors, FIELD_TYPE } from '@8base/utils';
+
+type IsTableFeaturesAllowedCommonArgs = {
+  tableName?: string;
+  appName?: string;
+  tableId?: string;
+};
+
+type IsFieldFeaturesAllowedCommonArgs = {
```

Comment:
@ej9x the same question

Reply:
I made the possibility to pass `tableId` with `fileId` or pass `tableName`, `appName`, `fileName`.

Reply:
U could make this possibility something like this:

```ts
type FooArgs = {
    id: string
    appName: string
} | {
    name: string
    appName: string
};

const foo = (args: FooArgs) => {
    console.log(args);
};

foo({ id: "id", appName: "appName "});
foo({ name: "name", appName: "appName "});
```